### PR TITLE
Clear validation states in reset methods

### LINKED ISSUE #2719: SSL Fields Stay in Error State Although Fields Reset
Although user resets all the fields or changes the views without saving changes, all the fields are marked red. 

**Testflow:**
1. Login to Kura as admin/admin
2. Go to Settings -> try to change anyt of the SSL fields
3. After you enter something in the fields, erase it and press Reset
4. Observe the fields that are marked in red

**Expected behavior**
After the Reset button is pressed, all the fields should be stripped of the red makration. 

**Screenshots**
Screenshot 2020-03-25 at 13 42 31

Target Environment (please complete the following information):
Board: Raspberry Pi 3
OS version: Linux raspberrypi 4.19.75-v7+ # 1270 SMP Tue Sep 24 18:45:11 BST 2019 armv7l GNU/Linux
Additional info: /

### PR DESCRIPTION
**Pull request by pierantoniomerlino**
Added clear of the validation state of GroupForms in reset method for the Settings tabs.

**Related Issue:** This PR fixes/closes #2719 

Signed-off-by: pierantoniomerlino <pierantonio.merlino@eurotech.com>

### CODE REVIEW

**Comment by MMaiero** [comment]
@mstankovic I have done a refactoring, can you please check that the QA tests are still passing?

**Comment by MMaiero** [comment]
@pierantoniomerlino Can you check if the changes I made are also ok for you?

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/settings/ServerCertsTabUi.ui.xml [hunk 1]
```diff
@@ -55,17 +55,19 @@
                     <b:FieldSet>
                         <b:FormGroup ui:field="groupStorageAliasForm">
                             <b:FormLabel for="storageAliasInput"
-                                ui:field="storageAliasLabel" />
+                                ui:field="storageAliasLabel" showRequiredIndicator="true"/>
```
Comment (LeoNerdoG) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/settings/ServerCertsTabUi.ui.xml:58]: Please update header to "2020".

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/settings/DeviceCertsTabUi.ui.xml [hunk 2]
```diff
@@ -54,25 +54,28 @@
                     <b:FieldSet>
                         <b:FormGroup ui:field="groupStorageAliasForm">
                             <b:FormLabel for="storageAliasInput"
-                                ui:field="storageAliasLabel" />
+                                ui:field="storageAliasLabel" showRequiredIndicator="true"/>
```
Comment (LeoNerdoG) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/settings/DeviceCertsTabUi.ui.xml:57]: Please update header to "2020".

Comment (MMaiero) [reply kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/settings/DeviceCertsTabUi.ui.xml:57]: Done

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/settings/ServerCertsTabUi.ui.xml (hunk 1 above)
Comment (MMaiero) [reply kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/settings/ServerCertsTabUi.ui.xml:58]: Done

**Comment by pierantoniomerlino** [comment]
@MMaiero For me it's ok!

**Pull request merged**